sw/airborne/modules/lidar/lidar_lite.c

```diff
@@ -62,7 +62,7 @@ void lidar_lite_init(void)
   lidar.distance = 0;
   lidar.distance_raw = 0;
 
-  init_median_filter(&lidar_filter);
+  init_median_filter(&lidar_filter, 7);
```

Comment:
This should be configurable from the airframe config file - maybe add a variable in lidar_lite.xml that specifies the size? Lets say in noisier applications I might want to change this value to something else.